vg/vgsvg/vgsvg.go

```diff
@@ -9,12 +9,13 @@ package vgsvg
 import (
 	"bufio"
 	"bytes"
-	"code.google.com/p/plotinum/vg"
 	"fmt"
-	svgo "github.com/ajstarks/svgo"
 	"image/color"
 	"io"
 	"math"
+
+	svgo "github.com/ajstarks/svgo"
```

Comment:
This has come up many times in the past. We need to add a comment explaining why we do this.